### DISCUSSION TITLE
generated type extend to entity type

### DISCUSSION THREAD

**Discussion by siggipop** [Power-Maverick/DataverseDevTools-VSCode#64] [Ideas]
How about if generated type was extended to be used with entity type as well as form context.
so that WebApi would be as easy as form context to develop with.

`let caseid: Xrm.LookupValue | undefined = _formContext.getAttribute("new_caseid").getValue()?.at(0);`
`let relatedcase: Xrm.New_case = await Xrm.WebApi.online.retrieveRecord(caseid.entityType, caseid.id);`
`let status = relatedcase.statuscode;`

**Comment by Power-Maverick** [comment]
Will look into it. Can you please submit this as a feature request under "Issues"? thank you.
